pkg/netid/netid.go

```diff
@@ -0,0 +1,61 @@
+package netid
+
+/*
+Accessor methods to annotate namespace for multitenant support
+*/
+
+import (
+	"fmt"
+	"math"
+	"strconv"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+)
+
+const (
+	// Current assigned VNID for the namespace
+	VNIDAnnotation string = "pod.network.openshift.io/multitenant.vnid"
+	// Desired VNID for the namespace
+	WantsVNIDAnnotation string = "pod.network.openshift.io/multitenant.wants-vnid"
```

Comment:
> why is there a "wants-vnid" and "vnid"? When is wants != vnid?
VNIDAnnotation represents the current network ID for the namespace and we want this annotation to be assigned only by the vnid controller (watches on namespace changes) to avoid any race conditions.
WantsVNIDAnnotation is intended to be used for requesting a particular vnid (example: oadm pod-network join-projects/make-projects-global). Vnid controller may satisfy this request if certain conditions are met (example: requested id already allocated, within defined range, etc.).